Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -46,9 +46,10 @@ static MTLLoadAction loadAction(WGPULoadOp loadOp)
         return MTLLoadActionLoad;
     case WGPULoadOp_Clear:
         return MTLLoadActionClear;
+    case WGPULoadOp_Undefined:
     case WGPULoadOp_Force32:
         ASSERT_NOT_REACHED();
-        return MTLLoadActionClear;
+        return MTLLoadActionDontCare;
```

Comment:
I'm kind of confused about this; it doesn't match the WebGPU spec IDL. Maybe I should open an issue on WebGPU.h about it....

Reply:
Yes I will do that, unclear why `WGPULoadOp_Undefined` was added

Reply:
@litherum as I was typing up the WebGPU issue I noticed the `_Undefined` categories exist for many (all?) of the other enums, it seems like it was an oversight not having it before